MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include CHANGELOG.rst
```

Comment:
There is no CHANGELOG.rst file, though there should be.

Reply:
I think I missed the memo that we are doing changelogs now. I see them in the cookiecutter but I don't know when that happened. It's not clear to me why we would ever do changelogs **and** github releases **and** conventional commits **and** PR templates. It seems like we're asking developers for a lot of overlapping information.

I'm going to just delete this line for now, but I'll poke arch-bom and pitch the idea of dropping one of those (hopefully GH releases) so we can focus on the things we really care about developers doing.

Reply:
I poked arch-bom: https://discuss.openedx.org/t/changelogs-and-reducing-developer-busywork/8435